statgen/pheget
# Change search behavior

## Ticket
Partly addresses #43

## Purpose
Add an API to return the best variant given a gene (symbol or gene_id), and rewrite/cleanup searchbox.js using Fetch API

## How to test this
Type different queries into the searchbox (chr:pos, chr:start-end, rsnum, gene_name, garbage input which should fail), and verify that the script is sending you to a page that makes sense:
- chr:pos -> single variant as requested
- chr:start-end -> range as requested
- rsnum -> single variant as requested
- gene_name -> **region view (with padding on both sides) for the tissue with the most significant eQTL** (updated)
- garbage input -> window alert of failure)

## Deployment / configuration notes
(none)
